lib/taskana-core/src/main/java/pro/taskana/TaskService.java

```diff
@@ -29,13 +31,18 @@
     Task claim(String id, String userName) throws TaskNotFoundException;
 
     /**
-     * Set task to completed.
-     * @param taskId
-     *            the task id
-     * @return changed Task after update.
-     * @throws TaskNotFoundException TODO
+     * Complete a claimed Task and update State and Timestamps.
+     *
+     * @param taskId - Id of the Task which should be completed.
+     * @param isForced - Flag which can complete a Task in every case if Task does exist.
+     *
+     * @return Task - updated task after completion.
+     *
+     * @throws TaskNotClaimedException when Task wasn´t claimed before.
+     * @throws TaskNotFoundException if the given Task can´t be found in DB.
+     * @throws NotOwnerException if current user is not the task-owner or administrator.
      */
-    Task complete(String taskId) throws TaskNotFoundException;
+    Task completeTask(String taskId, boolean isForced) throws TaskNotClaimedException, TaskNotFoundException, NotOwnerException;
```

Comment:
Should be seperated into **_completeTask(String taskId)_** and **_completeTask(String taskId, boolean isForced)_**

Reply:
Done